__tests__/app.test.js

```diff
@@ -43,21 +43,46 @@ describe("/api/topics", () => {
 describe("/api/articles", () => {
   describe("GET /api/articles/:articleid", () => {
     test("status: 200, responds with a single matching article", () => {
-      const articleid = 2;
+      const articleid = 1;
       const secondArticle = {
-        article_id: 2,
-        title: "Sony Vaio; or, The Laptop",
+        article_id: 1,
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: "2020-07-09T20:11:00.000Z",
+        votes: 100,
+      };
+      return request(app)
+        .get(`/api/articles/${articleid}`)
+        .expect(200)
+        .then((result) => {
+          expect(result.body.article).toEqual(
+            expect.objectContaining(secondArticle)
+          );
+        });
+    });
+    //test for left join
+  });
+  describe("GET /api/articles/:articleid COMMENT COUNT", () => {
+    test("status: 200, should return selected article, with a comment count", () => {
+      const articleid = 3;
+      const thirdArticle = {
+        article_id: 3,
+        title: "Eight pug gifs that remind me of mitch",
         topic: "mitch",
         author: "icellusedkars",
-        body: "Call me Mitchell. Some years ago—never mind how long precisely—having little or no money in my purse, and nothing particular to interest me on shore, I thought I would buy a laptop about a little and see the codey part of the world. It is a way I have of driving off the spleen and regulating the circulation. Whenever I find myself growing grim about the mouth; whenever it is a damp, drizzly November in my soul; whenever I find myself involuntarily pausing before coffin warehouses, and bringing up the rear of every funeral I meet; and especially whenever my hypos get such an upper hand of me, that it requires a strong moral principle to prevent me from deliberately stepping into the street, and methodically knocking people’s hats off—then, I account it high time to get to coding as soon as I can. This is my substitute for pistol and ball. With a philosophical flourish Cato throws himself upon his sword; I quietly take to the laptop. There is nothing surprising in this. If they but knew it, almost all men in their degree, some time or other, cherish very nearly the same feelings towards the the Vaio with me.",
-        created_at: "2020-10-16T05:03:00.000Z",
+        body: "some gifs",
+        created_at: "2020-11-03T09:12:00.000Z",
         votes: 0,
+        comment_counter: 2,
```

Comment:
refactor so that the key comes out as comment_count instead of comment_counter as stated on the trello